src/core/localization/translations/en/common.json

```diff
@@ -93,7 +93,34 @@
     "error_bad_layer_data": "Invalid MCDA layer data",
     "error_invalid_file": "Invalid MCDA file format",
     "legend_title": "Legend",
-    "legend_subtitle": "Hexagons are colored as weighted average of normalized and transformed layers values"
+    "legend_subtitle": "Hexagons are colored as weighted average of normalized and transformed layers values",
+    "layer_editor": {
+      "save_changes": "Save changes",
+      "range": "Range",
+      "outliers": "Outliers",
+      "sentiment": "Sentiment",
+      "weight": "Weight",
+      "transform": "Transform",
+      "transformation": "Transformation",
+      "normalize": "Normalize",
+      "normalization": "Normalization",
+      "reset_limits_to_default": "Reset limits to default",
+      "exclude": "Exclude",
+      "as_values_on_limits": "As values on limits",
+      "natural_logarithm": "Natural logarithm",
+      "square_root": "Square root",
+      "no": "No",
+      "max-min": "Max-min",
```

Comment:
Do we use snake_case for all keys?

Reply:
True, I'll replace the dash with underscore inside the key

Reply:
Fixed